.travis.yml

```diff
@@ -2,15 +2,15 @@ sudo: False
 
 language: python
 python:
-  - "3.8"
+  - "3.9"
```

Comment:
I don't think we want to pin it to 3.9.  If we don't pin, does it just run with the latest python?  I can see arguments both ways, but if we unpin at least our builds stop passing every time travis upgrades to the latest python.....

Reply:
I'm not familiar enough with how TravisCI works (I don't know if you can just say "give me a Python image"), but that number actually doesn't matter anyway as lower in the config, the setup steps involve downloading miniconda and setting up a new environment that the tests are run in, but that also has a pinned version in it. The solution you'd be looking for would be to have the conda create env statement say "python=3" instead of "python=3.x", but this obviously means we'd always be grabbing the latest version. Should I make that change and re-commit?